services/blockchain-app-registry/shared/metadataIndex.js

```diff
@@ -146,16 +144,14 @@ const deleteAppMeta = async (appMeta, dbTrx) => {
 const deleteTokensMeta = async (tokenMeta, dbTrx) => {
 	const tokenMetadataTable = await getTokenMetadataIndex();
 	await BluebirdPromise.map(
-		tokenMeta.localIDs,
-		async (localID) => {
+		tokenMeta.tokenIDs,
+		async (tokenID) => {
 			const queryParams = {
-				network: tokenMeta.network,
-				chainName: tokenMeta.chainName,
-				localID,
+				tokenID,
```

Comment:
Could we please ensure that `chainID` and `tokenID` are always in lower case when indexing them?

Reply:
Yes, since these are hex (`[0-9a-fA-F]`) representation of the values.